app/lib/hooks/useA11yErrorAnnouncement.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { AccessibilityInfo } from 'react-native';
+
+interface IUseA11yErrorAnnouncement {
+	errorMessage: string | undefined;
+}
+
+const useA11yErrorAnnouncement = ({ errorMessage }: IUseA11yErrorAnnouncement) => {
+	const [errorAnnounced, setErrorAnnounced] = useState(false);
```

Comment:
```suggestion
const useA11yErrorAnnouncement = ({ error }: IUseA11yErrorAnnouncement) => {
	const [announced, setAnnounced] = useState(false);
```

I'd rename for simplicity